plugins/quickinspector/quickoverlay.cpp

```diff
@@ -50,6 +50,50 @@ extern Q_GUI_EXPORT QImage qt_gl_read_framebuffer(const QSize &size,
 QT_END_NAMESPACE
 
 namespace GammaRay {
+
+class QQuickItemPropertyCache {
+public:
+    static const QQuickItemPropertyCache &getPropertyCache(QQuickItem *item)
+    {
+        // className() is generated by moc and static -> we can use the pointer as key
```

Comment:
As I understood @pasnox not all classes provide margins therefore we have to test for the class type.

Is any property cached a dynamic property?